pennylane/ops/qubit/arithmetic_ops.py

```diff
@@ -118,12 +118,34 @@ def _matrix(cls, *params):
         return QubitCarry._mat
 
     @staticmethod
-    def decomposition(wires):
-        decomp_ops = [
+    def compute_decomposition(wires):
+        """Determine ``QubitCarry``'s decomposition for specified parameters, wires,
```

Comment:
```suggestion
        """Compute the decomposition for specified parameters, wires,
```
Minor, and personal preference, but:

- I find ``CodeFormatting``<no space>[non code formatting] always a bit odd to read (and sometimes Sphinx will fail to render it!)
- I'm not sure there is a need to repeat the name of the class, since the class itself is assumed due to namespacing (at least, this matches the conventions of our other methods)

If this is accepted, you will need to make similar modifications in the other docstrings below

Reply:
I agree in principle, although I found it nicer to read the class repeated in the docstring. Plus, it is a lot of work to change...

Reply:
I'll be going through every docstring again anyway... 😜